File: aat-android/src/main/java/ch/bailu/aat/views/msg/overlay/InfoLogMsgView.java
package ch.bailu.aat.views.msg.overlay;

import android.content.Context;
import android.content.Intent;

import ch.bailu.aat.util.AppIntent;
import ch.bailu.aat.util.ui.ToolTip;
import ch.bailu.aat_lib.dispatcher.AppBroadcaster;

public class InfoLogMsgView extends AbsBroadcastMsgView {
    public InfoLogMsgView(Context context) {
        super(context, AppBroadcaster.LOG_INFO);
        ToolTip.themeify(this);
    }

    @Override
    public void set(Intent intent) {
        String message = intent.getStringExtra(AppIntent.EXTRA_MESSAGE);
        set(message);
    }
}

File: aat-android/src/main/kotlin/ch/bailu/aat/views/msg/overlay/InfoLogMsgView.kt
package ch.bailu.aat.views.msg.overlay

import android.content.Context
import android.content.Intent
import ch.bailu.aat.util.AppIntent
import ch.bailu.aat.util.ui.tooltip.ToolTip.themeify
import ch.bailu.aat_lib.broadcaster.AppBroadcaster

class InfoLogMsgView(context: Context) : AbsBroadcastMsgView(context, AppBroadcaster.LOG_INFO) {
    init {
        themeify(this)
    }

    override fun set(intent: Intent) {
        val message = intent.getStringExtra(AppIntent.EXTRA_MESSAGE)
        if (message is String) {
            set(message)
        }
    }
}
